### ISSUE TITLE
swapped sign of longitude

### ISSUE THREAD

**Issue by borje** [cpucycle/astrotime#1]
Isn't there a sign error somewhere? The sunrise and sunset is incorrect unless you swap the sign of the longitude. 
If you enter the coordinates from your example on (http://www.whatsmygps.com/) you won't end up in Washington...

**Comment by cpucycle** [comment]
You are correct. The code was ported from a c++ version, which was ported from the original java at http://www.esrl.noaa.gov/gmd/grad/solcalc/sunrise.html. That website has a message "Back when this calculator was first created, we decided to use a non-standard definition of longitude and time zone, to make coordinate entry less awkward. So on this page, both longitude and time zone are defined as positive to the west, instead of the international standard of positive to the east of the Prime Meridian." I think I can make it conform to the standard definition of longitude by simply negating longitude at the top of calcSunsetUTC and calcSunriseUTC.

**Comment by dinedal** [comment]
Any update on this?
